### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by lirui10093622** [getlantern/lantern-forum-cn#949] [General]
@Mike777Khoo , 你好，我购买专业版之后，账号一直没有激活，麻烦给我退一下款。我的邮箱：lirui10093622@163.com

我的支付账单：
![e1621adbe9fe831d78ddb8a0c3d55f6](https://github.com/user-attachments/assets/12f1c050-1e17-4505-87d8-359770e7e728)

**Comment by MaxMKMax** [comment]
@lirui10093622 您好. 非常抱歉影响了您的使用体验. 我们帮您激活好吗? 会发激活码给您. 先别退款好吗?

**Reply by lirui10093622** [reply]
我已经买了其他VPN产品了，麻烦给我退一下款吧，谢谢了

**Reply by lirui10093622** [reply]
@Mike777Khoo 我已经买了其他VPN产品了，麻烦给我退一下款吧，谢谢了

**Reply by MaxMKMax** [reply]
@lirui10093622 您好. 非常抱歉影响了您的体验感. 我帮您提交问题上去了.
希望您以后再回来时用蓝灯!
